### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by MartinEgli** [dotnet/reactive#1859] [General]
Hi all, what is the roadmap for Rx.net. 
It would be very good if it had the same cycle as .Net. 
How can we help?

**Comment by HowardvanRooijen** [comment]
There's a meeting with the parties interested in becoming maintainers, and the .NET Foundation happening in the next few weeks. 

A roadmap for 2023 (.NET 8.0 release train) and beyond was the 1st item in my suggested action plan.

I'll provide updates as soon as I have any!

**Comment by HowardvanRooijen** [comment]
🗺️Roadmap for v6 / v7 is available here: https://github.com/dotnet/reactive/discussions/1868
🚀 Rx.NET v6.0 for .NET 6.0 & .NET 7.0 is now GA: https://endjin.com/blog/2023/05/rx-dotnet-v6-released
